### PR TITLE
Fix aria-label warnings

### PR DESCRIPTION
**Pull request by joepadmiraal**
A lot of aria-label warnings are shown in the console while using paper-ui.
This makes it difficult to debug issues.
I added all missing aria-labels I could find.

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paperui/web/partials/control.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 			aria-label="Refresh"></md-button>
 	</div>
 	<md-tabs md-stretch-tabs="always" class="section-tabs"
-		md-selected="selectedTabIndex"> <md-tab
+		md-selected="selectedTabIndex" > <md-tab
```
Comment (sja) [inline extensions/ui/org.eclipse.smarthome.ui.paperui/web/partials/control.html]: Not needed space?

**Comment by sja** [comment]
Great! Thanks for that Contribution. I'll merge this in after the space is removed. I don't want to be pedantic, but it will keep merges smaller and less complicated.

**Comment by joepadmiraal** [comment]
Oops, removed the space.

**Comment by dnobel** [comment]
Yes, thank you very much! That helps a lot :)

**Pull request merged**